meta-resin-common/recipes-support/resin-mounts/resin-mounts/etc-docker.mount

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Docker key.json bind mount
-Requires=mnt-conf.mount
-After=mnt-conf.mount
+Requires=mnt-conf.mount resin-bind.target
```

Comment:
So that this type of services only start if if resin-bind strats, and if resin-bind starts it will start these services too.